### PR TITLE
[ISSUE #4559] Remove unnecessary operation of logicsMsgTimestamp

### PR DESCRIPTION
**Pull request by BurningCN**
1.Remove unnecessary operation of `logicsMsgTimestamp` (issue #4559)
2.When `DefaultMessageStore.this.getMessageStoreConfig().isDebugLockEnable() == true`, then need `scheduleAtFixedRate`.
3.Fix typo

### CODE REVIEW

**Review by tsunghanjacktsai — APPROVED** [review]
Pretty nice observation. It's really surprising that we've got more and more decent developers like you who paid attention to our codebase thoroughly and made it better.

**Review by juzi214032 — APPROVED** [review]

**Review by hzh0425 — DISMISSED** [review]

**Review by djl394922860 — DISMISSED** [review]

**Review by ni-ze — APPROVED** [review]

**Comment by ni-ze** [comment]
@BurningCN some conflicts happen in DefaultMessageStore, should merge from develop firstly.

**Review by tianliuliu — DISMISSED** [review]

File: store/src/main/java/org/apache/rocketmq/store/DefaultMessageStore.java [hunk 1]
```diff
@@ -1552,26 +1552,28 @@ public void run2() {
             }
         }, 1, 10, TimeUnit.MINUTES);
 
-        this.scheduledExecutorService.scheduleAtFixedRate(new AbstractBrokerRunnable(this.getBrokerIdentity()) {
-            @Override
-            public void run2() {
-                if (DefaultMessageStore.this.getMessageStoreConfig().isDebugLockEnable()) {
+        if (DefaultMessageStore.this.getMessageStoreConfig().isDebugLockEnable()) {
```
Comment (Oliverwqcwrw) [inline store/src/main/java/org/apache/rocketmq/store/DefaultMessageStore.java:1555]: I think this is  better  to put in the `run()`, can support the debugLockEnable dynamic modification, WDYT

Please let me know if I am missing something

Comment (BurningCN) [reply store/src/main/java/org/apache/rocketmq/store/DefaultMessageStore.java]: Thanks, It can indeed be dynamically updated

**Review by aooohan — APPROVED** [review]

**Review by ShannonDing — APPROVED** [review]

**Pull request closed without merging**